### PR TITLE
Implement Initial UI for Title Screen & More

### PR DESCRIPTION
**Pull request by LarsenEndeavors**
- Added background music for the title screen: `Heartbreaking(chosic.com).mp3`
- Added title screen image: `btcTitleScreen0.png`
- Created new scene for the title screen: `TitleScreen.tscn`
- Implemented title screen script: `TitleScreen.cs`

This commit sets up the initial user interface for the title screen, including background music and a title image.

#27 
#25 
#26 
#24 
#39

### CODE REVIEW

File: Scripts/UI/Title/TitleScreen.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        GetTree().ChangeSceneToFile("res://Scenes/Main.tscn");
+    }
+
+	private void OnExitPressed() 
+	{
+		GetTree().Quit();
+	}
+
+	// Called every frame. 'delta' is the elapsed time since the previous frame.
+	public override void _Process(double delta)
```
Comment (GoatEXE) [inline Scripts/UI/Title/TitleScreen.cs]: We should consider not using unused methods for readability (:

File: Scripts/UI/Title/TitleScreen.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+        // Play background music
+        musicPlayer.Play();
+        // Connect button signals
+        startButton.Connect("pressed", new Callable(this, nameof(OnNewGamePressed)));
+        exitButton.Connect("pressed", new Callable(this, nameof(OnExitPressed)));
+	}
+
+	private void OnNewGamePressed()
+    {
```
Comment (GoatEXE) [inline Scripts/UI/Title/TitleScreen.cs]: One more indent here would make this uniform with the rest of the class, which would be nice (:

Comment (LarsenEndeavors) [reply Scripts/UI/Title/TitleScreen.cs]: Removed, good call

**Review by GoatEXE — CHANGES_REQUESTED** [review]
I love your code (:

File: Scripts/UI/Title/TitleScreen.cs (hunk 2 above)
Comment (LarsenEndeavors) [reply Scripts/UI/Title/TitleScreen.cs]: Updated indenting... or did ? XD

**Pull request merged**